project/plugins.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.16.2")
+addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.17.0")
```

Comment:
I am not sure if this update can have consequences? First time wotrking with**Gitter 8**, is there any way I can test the changes?